docs/docs/administration/reverse-proxy.md

```diff
@@ -64,3 +64,52 @@ Below is an example config for Apache2 site configuration.
    ProxyPreserveHost On
 </VirtualHost>
 ```
+
+### Traefik Proxy example config
+
+The example below is for Traefik version 3.
+
+The most important is to increase the `respondingTimeouts` of the entrypoint used by immich. In this example of entrypoint `websecure` for port `443`. Per default it's set to 60s which leeds to videos stop uploading after 1 minute (Error Code 499). In the example it's set to 10 minutes. Hence video-uploads will fail after 10 minutes instead of one.
+
+`traefik.yaml`
+```yaml
+[...]
+entryPoints:
+  websecure:
+    address: :443
+    # this section needs to be added
+    transport:
+      respondingTimeouts:
+        readTimeout: 600s
+        writeTimeout: 600s
+        idleTimeout: 600s
```

Comment:
```suggestion
    transport:
      respondingTimeouts:
        readTimeout: 0
        idleTimeout: 0
```

Reply:
I think respondingTimeouts need to change as above. Value 0 is no timeout exists

Reply:
I commited it. But isn't it a bit of bad practice to never terminate a connection?

Reply:
I see this now. I think we should go back to 600s like the other proxy examples. I disagree with removing writeTimeout